pkgs/by-name/mc/mcdreforged/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  fetchFromGitHub,
+  python312Packages,
+  testers,
+  mcdreforged,
+}:
+
+python312Packages.buildPythonApplication rec {
+  pname = "mcdreforged";
+  version = "2.12.3";
+
+  src = fetchFromGitHub {
+    owner = "Fallen-Breath";
+    repo = "MCDReforged";
+    rev = "v${version}";
+    sha256 = "sha256-MLub++mkkB/jshpHJXtqgIhs7Gcb4jHUyHqGE65S8A8=";
+  };
+
+  dependencies = with python312Packages; [
+    colorlog
+    colorama
+    packaging
+    parse
+    prompt-toolkit
+    psutil
+    ruamel-yaml
+    typing-extensions
+  ];
+
+  nativeCheckInputs = with python312Packages; [ pytestCheckHook ];
+
+  passthru.tests = {
+    version = testers.testVersion { package = mcdreforged; };
+  };
+
+  meta = with lib; {
+    homepage = "https://mcdreforged.com";
+    description = "A rewritten version of MCDaemon, a python tool to control your Minecraft server.";
+    mainProgram = "mcdreforged";
+    license = licenses.lgpl3Plus;
```

```suggestion
    license = licenses.lgpl3Only;
```

https://github.com/Fallen-Breath/MCDReforged/blob/850e28fce8ad624fbdd9501f5a5faae3d3ef57f3/setup.py#L27

Didn't find a place saying "or later".